infra/README.md

```diff
@@ -1,3 +1,62 @@
+# Multichain Infrastructure Overview
+
+## Environments:
+- Testnet (Production)
+- Dev (Development)
```

Comment:
Why do we still have partner and testnet folders?

Reply:
Because partner is a codebase for our partners to use this code to deploy their environment. Testnet is our personal environment.